test/e2e/page-objects/pages/home/asset-list.ts

```diff
@@ -221,8 +221,9 @@ class AssetListPage {
 
   async importCustomTokenByChain(
     tokenAddress: string,
-    symbol: string,
+    symbol: string | undefined,
     chainId: string,
+    prefilledSymbol: boolean = false,
```

Comment:
:information_source:  the symbol can be prefilled when we try to import a token, in which we mock the request for its data, like the import-token test